src/services/effects.ts

```diff
@@ -46,8 +46,13 @@ export function withEffects(store: Store<Actions>) {
       chain(result)
         .zip<WriteProvidersResponse | WriteProvidersRequest>(providers)
         .partition(([res]: [WriteProvidersResponse]) => isWriteProvidersSuccessResponse(res))
-        .tap(([successes, errors]) =>
-          store.set('error')(`Failed to geocode ${errors.length} (of ${errors.length + successes.length}) providers`)
+        .tap(([successes, errors]) => {
+            if (errors.length > 0) {
+              store.set('error')(`Failed to geocode ${errors.length} (out of ${errors.length + successes.length}) providers`)
+            } else {
+              store.set('error')(`All ${successes.length} providers geocoded`)
```

Comment:
Is it possible to change the snack color as well?

Reply:
We should rename "error" to "message" (or something similar), and make it an object with a "type" (success | error) and a "text" (string) key.

Reply:
👍  > @pnbt